src/plugins/ceramic-feed/server/services/ceramic/CeramicClient.js

```diff
@@ -1,11 +1,14 @@
 const { assign, clone, toPairs } = require('lodash')
 const { basename } = require('path')
 const { URL } = require('url')
+const LocalStorage = require('localstorage-memory')
 
 const CeramicModel = require('./CeramicModel')
 const { metadata, filesystem } = require('../../utils')
 const { withArray } = require('../../utils/async')
 
+global.localStorage = LocalStorage //required for orbis
+
 class Post extends CeramicModel {
```

Comment:
we shouldn't refer to "ceramic" in the entities names, pls rename to OrbisModel / OrbisClient, also rename folders etc

Reply:
orbis is also based on ceramic. i've renamed to CeramicOrbisClient
the model is still only used in our custom ceramic client
and we still save to our custom impl until we depracate it